src/keycloak/chart/templates/istio-admin.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: keycloak-block-admin-access-from-public-gateway
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "keycloak.selectorLabels" . | nindent 6 }}
+  action: DENY
+  rules:
+  - to:
+    - operation:
+        paths: 
+          - "/admin*"
+          - "/realms/master*"
+          - "/metrics*"
+    when:
+    - key: source.namespace
+      notValues: ["istio-admin-gateway"]
+  - to:
+    - operation:
+        paths: 
+          # Never allow anonymous client registration except from the pepr-system namespace
+          # This is another fallback protection, as the KC policy already blocks it
+          - "/realms/{{ .Values.realm }}/clients-registrations/*"
+    when:
+    - key: source.namespace
+      notValues: ["pepr-system"]
```

This one's probably tricky to turn into an allowlist sadly.

I think this is equivalent to the current policy, I tested it locally:
```suggestion
    from:
    - source:
      notNamespaces: ["pepr-system"]
```